src/mmrelay/meshtastic/daemon_executor.py

```
@@ -0,0 +1,109 @@
+                    continue
+                try:
+                    result = fn(*args, **kwargs)
+                except BaseException as exc:  # Future must preserve task failures.
+                    future.set_exception(exc)
+                else:
+                    future.set_result(result)
+        finally:
+            with self._lock:
+                self._threads.discard(current)
+
+    def shutdown(self, wait: bool = True, *, cancel_futures: bool = False) -> None:
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)